sources/Outputs/src/Dyd.cpp

```diff
@@ -247,7 +247,6 @@ Dyd::writeMacroConnectors() {
 
   connector = dynamicdata::MacroConnectorFactory::newMacroConnector(macroConnectorGenSignalNName_);
   connector->addConnect("generator_N_value", "n_grp_@INDEX@_value");
-  connector->addConnect("generator_alphaSum_value", "alphaSum_grp_@INDEX@_value");
```

Comment:
This has no impacts on MAIN tests ?

Reply:
It doesn't. This alphSum variable was quite useless. The final result of the simulation is the same. The only change is that in the generator's equations, instead of having (N/alphaSum) we now have N (which has not the same value now of course).